Brauser.Common/Recipe/DryHop.cs

```diff
@@ -1,9 +1,14 @@
 namespace Brauser.Common.Recipe;
 
+/// <summary>
+/// While importing from MaischeMalzUndMehr.de it may be required to fill StartTime and Duration
+/// </summary>
 public class DryHop
 {
     public Guid Id { get; set; }
     public int? Order { get; set; }
     public string? Variety { get; set; }
     public string? Quantity { get; set; }
-}
+    public TimeSpan StartTime { get; set; }
```

Comment:
If it is a "Time", it should be of type `DateTimeOffset`, shouldn't it?

Reply:
Or is this more like "after X minutes" dry hop should be added?

Reply:
The second one. After some time you have to add stuff